WixSharp/Entities/Collection.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+
+namespace WixSharp.Entities
+{
+    public class Collection
+    {
+        /// <summary>
+        /// Collection ID (generated automatically by the catalog)
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Collection name
+        /// </summary>
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Media items(images, videos etc) associated with this collection
+        /// </summary>
+        [JsonProperty("media")]
+        public Media Media { get; set; }
+    }
+
+    public class CollectionResponse
```

Comment:
In the case when you only have 1 property in response, use `root` property in SDK to automatically resolve the correct data